Liturgy/ZZ_PrayersOfThanskgivingAfterCommunion/PrayersAfterCommunionText.tex

```diff
@@ -152,8 +152,8 @@ \section*{Troparion of the Patron (Saint Katherine)}
 Let us praise the all-lauded and noble bride of Christ,
 the godly Katherine,
 the guardian of Sinai and its defense,
-who is also our support and succour and our help;
-for with the Holy Spirit's power
+who is also our support and succor and our help;
+for with the Holy Spirit's sword
```

Comment:
Changed to match http://ww1.antiochian.org/katherine-apolytikion

Reply:
Also, it might be a good idea to switch to this apolytikion, since I originally found the one we've been using on the newbyz.org site which I believe comes from GOARCH translations.

Reply:
I agree, but I think this should be the last piece we introduce as a change for everyone.